IFC4x3/Properties/r/RequiredDisplayDistance_36D8aP$xH6CecE2OoCc7Dx/Documentation.md

```diff
@@ -0,0 +1 @@
+The maximum distance that has to be displayed. The value relates only to the signal type, not to the circumstances at a special position.
```

Comment:
The description does not match with the property name. How is it different to `MaximumDisplayDistance`?